atest/acceptance/keywords/print_page.robot

```diff
@@ -0,0 +1,18 @@
+*** Settings ***
+Documentation    Suite description
+Suite Setup       Go To Page "non_ascii.html"
+Resource          ../resource.robot
+
+*** Test Cases ***
+Print Page As PDF Without Print Options
+    Print Page As PDF
+
+Provide Print Options From Module

```

Comment:
With this test I just wanted to do a preliminary exploration of how to set print options. A while back for when one opens a browser there were ways to customize how the browser was started. To do this one had to do something similar, which is go into the python module and make an instance of the python class of options. Then add to that robot variable as class instance with a method on that class to add attributes. It was fairly ugly and the syntax was all python. But doable. Tatu really cleaned that up with just having a string of options. I wanted to see if one could add print options that old school way. Here I left in my failed attempts as a reminder of what failed and what worked so if we do explain it with this then I can say what and what not to do.